plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/TestBloomFilter.java

```diff
@@ -0,0 +1,114 @@
+import io.trino.spi.connector.ConnectorSession;
+import io.trino.spi.type.Type;
+import io.trino.spi.type.VarcharType;
+import org.apache.parquet.column.values.bloomfilter.BloomFilter;
+import org.apache.parquet.hadoop.metadata.ColumnChunkMetaData;
+import org.apache.parquet.hadoop.metadata.ParquetMetadata;
+import org.apache.parquet.io.api.Binary;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.util.Optional;
+import java.util.Set;
+
+import static com.google.common.collect.Iterables.getOnlyElement;
+import static io.trino.plugin.hive.HiveTestUtils.HDFS_ENVIRONMENT;
+import static io.trino.plugin.hive.HiveTestUtils.getHiveSession;
+import static io.trino.spi.type.VarcharType.createVarcharType;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+
+public class TestBloomFilter
+{
+    public static final ConnectorSession SESSION = getHiveSession(new HiveConfig().setHiveStorageFormat(HiveStorageFormat.PARQUET));
+
+    @Test
+    public void testReadBloomFilterParquet()
+            throws Exception
+    {
+        String[] testNames = {"hello", "parquet", "bloom", "filter"};
+        VarcharType columnType = createVarcharType(255);
+
+        File parquetFile = new File(Resources.getResource("bloom-filter.parquet").toURI());
+        TrinoFileSystemFactory fileSystemFactory = new HdfsFileSystemFactory(HDFS_ENVIRONMENT);
+        TrinoFileSystem fileSystem = fileSystemFactory.create(SESSION);
+        TrinoInputFile inputFile = fileSystem.newInputFile(parquetFile.getPath());
+
+        TrinoParquetDataSource dataSource = new TrinoParquetDataSource(inputFile, new ParquetReaderOptions(), new FileFormatDataSourceStats());
+        ParquetMetadata parquetMetadata = MetadataReader.readFooter(dataSource, Optional.empty());
+
+        ColumnChunkMetaData columnChunkMetaData = getOnlyElement(getOnlyElement(parquetMetadata.getBlocks()).getColumns());
+
+        BloomFilterStore bloomFilterStore = new BloomFilterStore(dataSource, getOnlyElement(parquetMetadata.getBlocks()), Set.of(columnChunkMetaData.getPath()));
+        Optional<BloomFilter> bloomFilterOptional = bloomFilterStore.readBloomFilter(columnChunkMetaData.getPath());
+        assertTrue(bloomFilterOptional.isPresent());
+        BloomFilter bloomfilter = bloomFilterOptional.get();
+
+        for (String testName : testNames) {
+            Slice testSlice = Slices.utf8Slice(testName);
+            long testHash = getBloomFilterHash(bloomfilter, testSlice, columnType);
+            assertTrue(bloomfilter.findHash(testHash));
+        }
+
+        Slice testSlice = Slices.utf8Slice("not_exist");
+        long testHash = getBloomFilterHash(bloomfilter, testSlice, columnType);
+        assertFalse(bloomfilter.findHash(testHash));
+    }
+
+    @Test
+    public void testReadBloomFilterParquetNonExist()
+            throws Exception
+    {
+        File parquetFile = new File(Resources.getResource("no-bloom-filter.parquet").toURI());
```

Comment:
Similar comment as above about using `ParquetTester#writeParquetColumn` instead of static file